template_helpers/templatetags/template_helpers.py

```diff
@@ -38,3 +47,114 @@ def split(value, sep=' '):
 
     """
     return value.split(sep)
+
+
+@register.filter
+@stringfilter
+def starspan(value):
+    """
+    The ``starspan`` template filter adds extra span element to star
+    indicated text ("***text***").
+
+    .. code-block:: text
+
+        {{ some_text_variable|starspan }}
+
+    """
+    return mark_safe(STARSPAN_RE.sub(r'<span>\2</span>', conditional_escape(value)))
+
+
+@register.filter
+def merge_lists(value, list_to_merge):
```

Comment:
I'd rename the filter to "merge_list" as we merge only one list and not multiple.